src/frontend/screens/Observation/PhotoPreviewModal.tsx

```diff
@@ -27,17 +71,75 @@ export const PhotoPreviewModal: FC<
           onPress={() => navigation.goBack()}
         />
       ),
+      // eslint-disable-next-line react/no-unstable-nested-components -- it's correct syntax
+      headerRight: () =>
+        deletable ? (
+          <Pressable onPress={openSheet} style={styles.deleteButtonWrapper}>
+            <MaterialIcons name="delete" size={18} color={WHITE} />
+            <Text style={styles.deleteButtonText}>
+              {t(m.headerDeleteButtonText)}
+            </Text>
+          </Pressable>
+        ) : (
+          <></>
+        ),
     });
   });
 
   return (
     <>
       <StatusBar style="light" />
-      <PhotoUnpreparedView
-        observationId={observationId}
-        attachmentId={attachmentId}
-        variant={'original'}
-      />
+      {observationId && attachmentId ? (
+        <PhotoUnpreparedView
+          onPress={() => setShowHeader(!showHeader)}
+          observationId={observationId}
+          attachmentId={attachmentId}
+          variant={'original'}
+        />
+      ) : originalPhotoUri ? (
+        <PhotoPreparedView photoUri={originalPhotoUri} />
+      ) : (
+        <></>
+      )}
+
+      <BottomSheetModal ref={sheetRef} isOpen={isOpen}>
+        <BottomSheetContent
+          title={t(m.deleteModalTitle)}
+          buttonConfigs={[
+            {
+              text: t(m.deleteModalDeleteButton),
+              dangerous: true,
+              onPress: handlePhotoDelete,
+              variation: 'filled',
+              icon: <MaterialIcons name="delete" size={24} color={WHITE} />,
+            },
+            {
+              text: t(m.deleteModalCancelButton),
+              onPress: closeSheet,
+              variation: 'outlined',
+            },
+          ]}
+        />
+      </BottomSheetModal>
     </>
   );
 };
+
+const styles = StyleSheet.create({
+  deleteButtonWrapper: {
+    borderStyle: 'solid',
+    borderWidth: 1,
+    borderColor: WHITE,
+    borderRadius: 3,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingHorizontal: 4,
+    paddingVertical: 6,
```

Comment:
Can you make the button more similar to the button component. Rounded edges mainly